frontend/src/utils/APIClient.js

```diff
@@ -1,5 +1,5 @@
 import {
-    SPONSORS_URL
+    SPONSORS_URL, EVENT_URL as EVENTS_URL
```

Comment:
Not a big thing, but maybe rename it to `EVENTS_URL` from Constants.js to avoid having to rename it here.

Reply:
Whoops, I suspect my editor did that when I misnamed a variable.